src/Aquifer.API/Endpoints/Feedback/Resources/Content/Create/Request.cs

```diff
@@ -9,5 +9,6 @@ public record Request
     public string? ContactValue { get; set; }
     public FeedbackContactType? ContactType { get; set; }
     public string? Feedback { get; set; }
+    public FeedbackType FeedbackType { get; set; }
```

Comment:
I think we should default this to whatever one the existing feedback form will go to that way old clients sending data aren't setting it to `None`

Reply:
It gets so little traffic I think it's okay not to put in a default that isn't really desired (we only got our first feedback today actually). But that said, the validator's gonna fail for an old client with the `NotEmpty()`, so I think that needs to be removed for now.

Reply:
I'm putting in FeedbackType for the existing feedback call in well. Maybe I can put the validator back in once that's deployed